packages/nitro/src/rollup/plugins/externals.ts

```
@@ -87,7 +87,7 @@ export function externals (opts: NodeExternalsOptions): Plugin {
 
         const writeFile = async (file) => {
           // Skip symlinks that are included in fileList
-          if (await fse.stat(file).then(i => i.isDirectory())) {
+          if (await fse.stat(file).then(i => i.isDirectory()).catch(() => true)) {
```

We can probably make a helper function for this. `err.code` need to be tested against `ENOENT` and I guess we an only exclude `package.json` as safe-to-ignore. Generally hiding errors is probably not best idea and covers other bugs

---

fee8ed4978444d16a3baa9e66043072e08ceb94e